yarn.lock

```diff
@@ -6560,7 +6560,7 @@ path-type@^3.0.0:
   dependencies:
     pify "^3.0.0"
 
-pbkdf2@^3.0.16, pbkdf2@^3.0.3, pbkdf2@^3.0.9:
+pbkdf2@^3.0.3, pbkdf2@^3.0.9:
```

Comment:
We could get rid of the two pbkdf2 installations by removing this block and letting yarn repair it with `yarn install`